src/main/js/controls/Graph/helpers/helpers.js

```diff
@@ -87,6 +87,32 @@ const translateDefs = (config, canvasSVG) =>
             constants.Y_AXIS,
             config.axis.x.orientation && calculateVerticalPadding(config)
         );
+
+    if (
+        config.settingsDictionary.shouldCreateDatelineDefs &&
+        config.dateline.length > 0
+    ) {
+        const shapeHeightArr = [];
+        const shape = d3.selectAll(`.${styles.datelinePoint}`);
+        shape[0].forEach((element) => {
+            const shapeHeight = element.getBBox().height;
```

Comment:
Doing a `getBBox` will cause a repaint on the page. Since this is called on every resize, it would lead to repaint everytime. 
Can we do this just once and then reuse? The values are not going to change are they?

Reply:
I will place the calculation of the height of dateline indicator in dateline.js and import it where required